### PR TITLE
feat(@spartacus/core): add meta loader util into storefinder store

### LINKED ISSUE #475: Load once in the store, in storefinder [after 465]
Do this one after #465.

Originating from #272 and discussed in Nov 29 refinement.

Only load once in the store.
Not every module needs it. Close if not with a comment.
To be done after epic/accessors>functions

### PR DESCRIPTION
**Pull request by Matejk00**
Closes #475

### CODE REVIEW

File: projects/core/src/store-finder/store/selectors/find-stores.selectors.ts [hunk 1]
```diff
@@ -1,29 +1,36 @@
...
-  fromFeatureSelector.getStoreFinderState,
-  (state: StoresState) => state.findStores
+  getStoreFinderState,
+  (storesState: StoresState) => storesState.findStores
 );
 
 export const getFindStoresEntities: MemoizedSelector<
-  StoresState,
+  StateWithStoreFinder,
   any
```
Comment (kacperknapik) [inline projects/core/src/store-finder/store/selectors/find-stores.selectors.ts]: can you please make sure not to have any `any` types?

File: projects/core/src/store-finder/store/selectors/view-all-stores.selectors.ts [hunk 2]
```diff
@@ -1,29 +1,36 @@
...
-  fromFeatureSelector.getStoreFinderState,
-  (state: StoresState) => state.viewAllStores
+  getStoreFinderState,
+  (storesState: StoresState) => storesState.viewAllStores
 );
 
 export const getViewAllStoresEntities: MemoizedSelector<
-  any,
+  StateWithStoreFinder,
   any
```
Comment (kacperknapik) [inline projects/core/src/store-finder/store/selectors/view-all-stores.selectors.ts]: please get rid of `any` types

File: projects/core/src/store-finder/store/store-finder-state.ts [hunk 3]
```diff
@@ -1,16 +1,21 @@
...
 
 export interface StoresState {
-  findStores: FindStoresState;
-  viewAllStores: ViewAllStoresState;
+  findStores: LoaderState<FindStoresState>;
+  viewAllStores: LoaderState<ViewAllStoresState>;
 }
 
 export interface FindStoresState {
   findStoresEntities: any;
```
Comment (kacperknapik) [inline projects/core/src/store-finder/store/store-finder-state.ts]: please get rid of `any` types

File: projects/core/src/store-finder/store/store-finder-state.ts [hunk 4]
```diff
@@ -1,16 +1,21 @@
...
-  isLoading: boolean;
 }
 
 export interface ViewAllStoresState {
   viewAllStoresEntities: any;
```
Comment (kacperknapik) [inline projects/core/src/store-finder/store/store-finder-state.ts]: please get rid of `any` types

File: projects/core/src/store-finder/store/selectors/find-stores.selectors.ts (hunk 1 above)
Comment (Matejk00) [reply projects/core/src/store-finder/store/selectors/find-stores.selectors.ts]: Done.

File: projects/core/src/store-finder/store/selectors/view-all-stores.selectors.ts (hunk 2 above)
Comment (Matejk00) [reply projects/core/src/store-finder/store/selectors/view-all-stores.selectors.ts]: Done.

File: projects/core/src/store-finder/store/store-finder-state.ts (hunk 4 above)
Comment (Matejk00) [reply projects/core/src/store-finder/store/store-finder-state.ts]: Done.

**Pull request merged**